datasource/src/main/resources/liquibase/changelog/changelog-0.1.xml

```diff
@@ -65,4 +65,25 @@
         </rollback>
     </changeSet>
 
+    <changeSet id="create_run_table" author="manuel_vigueras">
+        <createTable tableName="run">
+            <column name="run_id" type="${uuid.type}"/>
+            <column name="project_id" type="${uuid.type}"/>
+            <column name="project_version" type="${text.type}"/>
+            <column name="status" type="${text.type}"/>
+            <column name="time_start" type="${timestamp.type}"/>
+            <column name="time_end" type="${timestamp.type}"/>
+            <column name="user_id" type="${uuid.type}"/>
+            <column name="results" type="${text.type}"/>
+            <column name="cmwl_workflow_id" type="${text.type}"  defaultValueBoolean="true"/>
```

Comment:
Better to remove this strange default value.
```suggestion
            <column name="cmwl_workflow_id" type="${text.type}"/>
```

Sorry, I just noticed this.

Reply:
Awesome catch! Thanks Nikita! Changed it